### DISCUSSION TITLE
Select does not work when combined with Radio in Chakra-UI

### DISCUSSION THREAD

**Discussion by hhimanshu** [chakra-ui/chakra-ui#4591] [Q&A] [answered]
I have a design where I need to combine radio group and select items together. Specifically, when a user select an option, then need to select an item from the list.

My implementation at https://codesandbox.io/s/chakra-radio-select-cv6r0?file=/src/index.tsx shows how it should look.

However, when I click on the list to select from drop down, nothing happens.

The select works independently, but not when nested inside RadioGroup. This is reproducible on [codesandbox](https://codesandbox.io/s/chakra-radio-select-cv6r0?file=/src/index.tsx)

Could someone help me understand what is incorrect here and what I need to do to be able to select the item from the list?

Thanks

**Comment by payapula** [comment, marked as answer]
@hhimanshu  I have played around in the sandbox. 

Strange thing is, we can focus on Select with `tab` keys and then select the option with our arrow keys, but it's not allowing any pointer events on `Select`.

I guess this is because when you use it inside Radio, the whole area is taken for Radio selection.  

![image](https://user-images.githubusercontent.com/7134153/129946172-7823d12f-3a78-4427-935d-ca8ee07f5421.png)

I mean wherever you click in the rectangle highlighted portion, the Radio is selected, except on the `Select` component. The workaround that I suggest is to move `Select` out of `Radio`, like this https://codesandbox.io/s/chakra-radio-select-forked-b94ki?file=/src/index.tsx

```jsx
<Flex alignItems={"center"}>
    <Radio value={"LIST"}>
       <Text ml={2}>List</Text>
    </Radio>
    <Select
      w="unset"
      ml={4}
      placeholder="Select List"
      onChange={(val) => setValue(val.target.value)}
      value={value}
    >
        <option value="option1">Option 1</option>
        <option value="option2">Option 2</option>
        <option value="option3">Option 3</option>
    </Select>
</Flex>
```

By this way, we are still keeping `Select` inside `RadioGroup`.

Let me know if this works for you.

**Reply by hhimanshu** [reply]
Hey @payapula , thanks for the help. Yes, that works
